src/dotnet-interactive-vscode/package.json

```diff
@@ -45,6 +45,26 @@
         ]
       }
     ],
+    "configuration": {
+      "title": ".NET Interactive Notebook",
+      "properties": {
+        "dotnet-interactive.interactiveToolSource": {
+          "type": "string",
```

Comment:
This is good but even just "use latest" or somethign along that line to switch from nuget.org to this feed